x-pack/plugins/upgrade_assistant/public/application/components/overview/fix_logs_step/fix_logs_step.tsx

```diff
@@ -47,13 +49,37 @@ const i18nTexts = {
         'Go to your logs directory to view the deprecation logs or enable log collecting to see them in the UI.',
     }
   ),
+  deniedPrivilegeTitle: i18n.translate(
+    'xpack.upgradeAssistant.overview.deprecationLogs.deniedPrivilegeTitle',
+    {
+      defaultMessage: `You're missing index privileges`,
+    }
+  ),
+  deniedPrivilegeDescription: (privilegesMissing: MissingPrivileges) => (
+    <FormattedMessage
+      id="xpack.upgradeAssistant.overview.deprecationLogs.deniedPrivilegeDescription"
+      defaultMessage="To be able to analyze deprecation logs, you must have {privilegesCount, plural, one {this index privilege} other {these index privileges}}: {missingPrivileges}"
```

Comment:
This doesn't read quite right to me. `.logs-deprecation.elasticsearch-default` is an _index_, not an _index privilege_. It's not clear what specific index privilege you need.

In my testing, I was able to get it working with the `all` index privilege on `.logs-deprecation.elasticsearch-default`. However, that's pretty permissive. We should confirm there's not another combination that'll work.

I'd change the copy to something like:

```
To analyze deprecation logs, you need the <x> index privilege{s} for `<index-name>`.
```

Reply:
Thanks for having a look @jrodewig! The minimum required privilege for the index is [read](https://github.com/elastic/kibana/pull/112420/files#diff-11b43fbb37f161529fef857c3eb295e8241be3f971a598c7e397d8d8807311bdR54), I've updated the PR description to reflect that also.

 I've also updated the copy of the callouts to reflect the changes you suggested:
![Screenshot 2021-09-21 at 12 19 32](https://user-images.githubusercontent.com/1191206/134154271-815e99e2-2b83-4e72-b58c-80855eef6497.png)
![Screenshot 2021-09-21 at 12 19 35](https://user-images.githubusercontent.com/1191206/134154274-5537d36a-1d8c-4124-9d7f-0291d7de0017.png)